.build/check-broken-links.sh

```diff
@@ -50,12 +53,20 @@ export PREVIEW_MODE=1
 npm run spin >/dev/null 2>&1 &
 
 ## wait for portal to be up and running
-timeout 60s bash -c 'until curl --silent -f http://localhost:3000 > /dev/null; do sleep 2; done'
+timeout 10s bash -c 'until curl --silent -f http://localhost:3000 > /dev/null; do sleep 2; done'
+
+echo "checking website health"
+
+resp_code=$(curl -o /dev/null -s -w "%{http_code}\n" 127.0.0.1:3000)
+[[ "${resp_code}" == "200" ]] && \
+  (echo "Success: website returned a 200 response code") || \
+  (echo "Failure: unexpected response code: ${resp_code}" && exit 1)
```

Comment:
Just have a curious question. would we not be able to get the response code from the curl while we wait for the website to come up?

Reply:
Good point, with the `-f` flag above, it will just timeout if the site doesn't return a successful response code.  We could leave that as is and continue to bubble up generic "timeout exceeded waiting for the site to come up" _or_ we might consider removing the `-f` flag and proceeding with this check and fail with an error mentioning the specific non-200 error code being returned.  WDYT?